travis/pg_travis_multi_test

```diff
@@ -12,16 +12,33 @@ fi
 status=0
 testtargets="$*"
 
-# Configure, build, and install extension
-./configure --enable-coverage PG_CONFIG=/usr/lib/postgresql/$PGVERSION/bin/pg_config
+# If USE_CUSTOM_PG_BUILD variable is not set, we will assume custom build is
+# not required. If custom build is not required, that means PostgreSQL was
+# installed via apt, otherwise we compiled it from the source. Depending
+# on the situation, location of pg_config is different.
+if [ "${USE_CUSTOM_PG_BUILD-0}" = "0" ]; then
+  PG_CONFIG=/usr/lib/postgresql/$PGVERSION/bin/pg_config
+else
+  PG_CONFIG=/usr/local/pgsql/bin/pg_config
+fi
+
+./configure --enable-coverage PG_CONFIG=$PG_CONFIG
+
+# Build, and install extension
 make all
 sudo make install
 
 # Change to test directory for remainder
 cd src/test/regress
 
-# Run tests. DBs owned by non-standard owner put socket in /tmp
-make ${testtargets} || status=$?
+# If USE_CUSTOM_PG_BUILD variable is not set or set to false we will only run
+# testtargets, otherwise we will run check-vanilla and check isolation along
+# with our regular regression tests.
+if [ "${USE_CUSTOM_PG_BUILD-0}" = "0" ]; then
+  make ${testtargets} || status=$?
+else
+  make ${testtargets} && make check-vanilla && make check-isolation || status=$?
```

Comment:
Don't call make three times here. In fact, it would be easiest to just check if `USER_CUSTOM_PG_BUILD` is set, then add `check-vanilla` and `check-isolation` to `testtargets`. Then you can use the existing invocation of `make` to do what you want.